### ISSUE TITLE
arm/stm32l4-discovery is broken

### ISSUE THREAD

**Issue by anton-bondarev** [embox/embox#2039]

**Comment by puranjaymohan** [comment]
@anton-bondarev is this problem is due to the template or the BSP? I tried changing some paths in the BSP's Mybuild and arm/stm32l4-discovery built correctly.
I changed all references of `STM32Cube_FW_L4_V1.14.0` to `STM32CubeL4-1.14.0` in every place in the `embox/third-party/bsp/stml4cube/Mybuild `and after this it built properly.

**Comment by anton-bondarev** [comment]
> I changed all references of `STM32Cube_FW_L4_V1.14.0` to `STM32CubeL4-1.14.0` in every place in the `embox/third-party/bsp/stml4cube/Mybuild `and after this it built properly.

@puranjaymohan Great. Add a commit with the changes to PR #2038
And you close both issues  #2039 and #2035

**Comment by puranjaymohan** [comment]
@anton-bondarev I have made the changes and added the commit.
Please merge the PR, when you feel it is correct.

**Comment by puranjaymohan** [comment]
@anton-bondarev I was trying to build this and an error started to appear 
```error: 'OPTION_NUMBER_embox__driver__serial__stm_usart_f3__usart1_irq' undeclared here (not in a function); did you mean 'OPTION_NUMBER_embox__driver__serial__stm_usart_l4__usart1_irq'?```

What is this? how to solve this?

**Comment by anton-bondarev** [comment]
@puranjaymohan Give me please more detailed log
`OPTION_NUMBER_embox__driver__serial__stm_usart_f3__usart1_irq`
It seems you've included 'stm_usart_f3' module instead of 'stm_usart_l4'

**Comment by anton-bondarev** [comment]
@puranjaymohan Ops, I see.

**Comment by puranjaymohan** [comment]
This line is in the header stm32_usart_conf_l4.h
```#define USART1_IRQ    \
     OPTION_MODULE_GET(embox__driver__serial__stm_usart_f3, NUMBER, usart1_irq)

**Comment by puranjaymohan** [comment]
Why is this here, also earlier it was building fine, I don't know what happened now. Also, I didn't add that line there.

**Comment by anton-bondarev** [comment]
```
diff --git a/src/drivers/serial/stm32_usart/stm32_usart_conf_l4.h b/src/drivers/serial/stm32_usart/stm32_usart_conf_l4.h
index 24dc6c3db..b5ea09df2 100644
--- a/src/drivers/serial/stm32_usart/stm32_usart_conf_l4.h
+++ b/src/drivers/serial/stm32_usart/stm32_usart_conf_l4.h
@@ -25,7 +25,7 @@
 #define MODOPS_USARTX OPTION_GET(NUMBER, usartx)
 
 #define USART1_IRQ    \
-       OPTION_MODULE_GET(embox__driver__serial__stm_usart_f3, NUMBER, usart1_irq)
+       OPTION_MODULE_GET(embox__driver__serial__stm_usart_l4, NUMBER, usart1_irq)
 static_assert(USART1_IRQ == USART1_IRQn);
 
 #if MODOPS_USARTX == 1
```
Yes It's not your fault It happend in #2033

**Comment by puranjaymohan** [comment]
@anton-bondarev So should, I add another PR for this or it will be solved by editing #2033

**Comment by anton-bondarev** [comment]
> @anton-bondarev So should, I add another PR for this or it will be solved by editing #2033

Thanks a lot!

**Comment by puranjaymohan** [comment]
@anton-bondarev I have added a PR for solving this, Now I can work on porting it to Nucleo Board :smile_cat:
